mailpoet/lib/EmailEditor/Integrations/Core/Renderer/Blocks/Image.php

```diff
@@ -137,4 +163,33 @@ private function addStyleToElement($blockContent, array $tag, string $style): st
 
     return $blockContent;
   }
+
+  /**
+   * @param string $blockContent
+   * @return array{imageUrl: string, image: string, caption: string}|null
+   */
+  private function parseBlockContent(string $blockContent): ?array {
+    // Suppress warnings for invalid HTML tags
+    libxml_use_internal_errors(true);
+    $dom = new \DOMDocument();
+    $dom->loadHTML($blockContent);
```

Comment:
This will cause an error when $blockContent is empty. To replicate this error, you can insert an image block, don't pick any image, save the document, and open the preview in a new tab.
![Screenshot 2024-01-10 at 13 26 25](https://github.com/mailpoet/mailpoet/assets/1082140/1c90276c-2456-4225-bc8b-9ae70c9a394c)